Language/Mixins/Mixin_Classes/syntax_A04_t02.dart

```diff
@@ -38,7 +38,7 @@ class C<T> extends S with M {
 
 main() {
   C c = new C();
-  c.m1.expectStaticType<Exactly<num?>>();
-  c.m2.expectStaticType<Exactly<String?>>();
-  c.s.expectStaticType<Exactly<Object?>>();
+  c.m1?.expectStaticType<Exactly<num>>();
+  c.m2?.expectStaticType<Exactly<String>>();
+  c.s?.expectStaticType<Exactly<Object>>();
```

Comment:
This is confusing: `C` means `C<dynamic>`, but that doesn't actually matter because we will get an `S<dynamic>` and an `M<num, String>` as superinterfaces no matter what (because of the raw types used as superinterfaces), and this means that `c.m1` and `c.m2` will have types `num` and `String`, but `c.s` will have the type `dynamic`.

So we can't execute an extension method on `c.s`, but when we're using `c.s?.expect...` we will avoid trying to call anything, because `c.s == null`.

So the `?` in `c.s?.` is crucial, but in `c.m1?.` and in `c.m2?.` it should be unnecessary.

I actually think it would be nicer to test this using `if (2 < 1) c.s.whatever;` as usual, and then keep the old forms in line 41 and 42.